opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -429,9 +423,7 @@ def get_label_set(self, labels: Dict[str, str]):
 
 
 class MeterProvider(metrics_api.MeterProvider):
-    def __init__(
-        self, resource: Resource = Resource.create_empty(),
-    ):
+    def __init__(self, resource: Resource = Resource.create_empty()):
```

Comment:
Out of curiosity, did you catch this manually or lint complained?

Reply:
Lint complained :)